packages/model/src/research-output.ts

```diff
@@ -221,9 +221,13 @@ export type ResearchOutputCreateDataObject = ResearchOutputCoreObject & {
   organismIds: string[];
   subtypeId?: string;
   teamIds: string[];
+  addedDate?: string;
```

Comment:
i wonder if we really want this to be optional? do we want the clients to have the option to change it if necessary? (im just asking, its possible that we do)

Reply:
That's an important decision to make as this type is used in many places/tests (data-providers and controllers, for its inputs/outputs/queries/responses)

Reply:
ok i think i got it wrong, this is the type for the data-provider. However I can see that the addedDate is in the ResearchOutputCoreObject and it should stay required as is. 

All you are doing in this PR is removing the `addedDate` from the Post request (and therefore type) and making sure that the controller fills in the blank